pttrack/tests.py

```diff
@@ -225,10 +225,17 @@ def test_document_urls(self):
             author_type=models.ProviderType.objects.all()[0])
 
         p = models.Document.objects.get(id=1).image.path
+        random_name = p.split("/")[-1]
```

Comment:
The most pythonic way to do this is `os.path.basename`. Not super important here, but can save you some annoyances down the line.